elide-integration-tests/src/test/java/com/yahoo/elide/tests/SortingIT.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.tests;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.restassured.RestAssured;
+import com.yahoo.elide.initialization.IntegrationTest;
```

Comment:
The imports will fail checkstyles.  Static at the top, followed by elide packages, followed by non java.util, followed by java util (all alphabetic within each section).